packages/react-native/src/primitives/Label/Label.tsx

```diff
@@ -9,7 +9,7 @@ export default function Label({
   ...rest
 }: LabelProps): JSX.Element {
   return (
-    <Text {...rest} style={[styles.label, style]}>
+    <Text {...rest} accessibilityRole="text" style={[styles.label, style]}>
```

Comment:
👍  Let's make sure we `Omit` the `accessibilityRole` on the `LabelProps` since it's not configurable